app/views/public/api/v1/portals/articles/show.html.erb

```diff
@@ -12,25 +12,25 @@
 <% end %>
 
 <% if !@is_plain_layout_enabled %>
-<div class="bg-woot-50 py-8 shadow-inner" style="background: url(/assets/images/hc/grid.svg) color-mix(in srgb, <%= @portal.color %> 10%, white)">
+<div class="bg-woot-50 dark:bg-woot-900 py-8 shadow-inner" style="background: url(/assets/images/hc/grid.svg) color-mix(in srgb, <%= @portal.color %> 10%, <%= @theme == 'dark' ? 'black' : 'white' %>)">
```

Comment:
Please create a dark variant of the grid and switch it if the color scheme is dark.

Reply:
👍